include/deal.II/fe/fe.h

```diff
@@ -1574,6 +1574,23 @@ class FiniteElement : public Subscriptor,
   element_multiplicity (const unsigned int index) const;
 
   /**
+    * Return a reference to a child finite element that matches the components
+    * selected by the given ComponentMask @p mask.
+    *
+    * For an arbitrarily nested FESystem, this function returns the inner-most
+    * FiniteElement that matches the given mask. The method fails if the @p mask
+    * does not exactly match one of the contained finite elements. It is most
+    * useful if the current object is an FESystem, as the return value can
+    * only be @p this in all other cases.
```

Comment:
I think it would be nice to have examples here such as those that you have in the issue you are closing with this PR.